components/publish/KeepNote.tsx

```diff
@@ -1,5 +1,6 @@
 import styled from "styled-components"
 import { useState } from "react"
+import { Link } from "react-admin"
```

Comment:
nit: Is this the right import?

Reply:
not sure where that came from, we generally would use Internal here so I went and subbed that in